.github/workflows/release.yml

```diff
@@ -6,8 +6,14 @@ name: Release
       - next
       - beta
       - "*.x"
+permissions:
+  contents: read  #  for checkout
 jobs:
   release:
+    permissions:
+      contents: write       #  to be able to publish a GitHub release
+      issues: write         #  to be able to comment on released issues
+      pull-requests: write  #  to be able to comment on released pull requests
```

Comment:
why split up permissions, there is only one job. I'd combine the permissions all under the top level `permissions` key

Reply:
okidoke, apologies, totally makes sense, I missed Matt's message